moped-editor/src/views/projects/signalProjectTable/TaskOrderAutocomplete.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { CircularProgress } from "@material-ui/core";
+import { Autocomplete, Alert } from "@material-ui/lab";
+import { useSocrataJson } from "src/utils/socrataHelpers";
+import {
+  TaskOrderInput,
+  SOCRATA_ENDPOINT,
+} from "src/utils/taskOrderComponentHelpers";
+import { filterOptions } from "src/utils/autocompleteHelpers";
+
+/**
+ * Material Autocomplete wrapper that enables selecting a task order record from a
+ * Socrata dataset and setting it.
+ * * @param {Object} classes - MaterialUI style object
+ * * @param {Object} props - passes down props parent MUI Table
+ * * @param {Array} value - passes an array of objects, each representing a task order
```

Comment:
might be good to document here what the shape of each task order {Object} looks like. ie what properties it should have.